pkg/cli/admin/catalog/mirror.go

```diff
@@ -585,6 +590,20 @@ func aggregateICSPs(icsps [][]byte) []byte {
 	return aggregation
 }
 
+func (o *MirrorCatalogOptions) PostRun() error {
```

Comment:
Does this method need to be exported anymore?

Reply:
Nope, good point.